package com.github.catomizer.error

open class BaseException(cause: Throwable? = null) : RuntimeException(cause)
